encryption_works.md

```diff
@@ -96,6 +96,78 @@ It's important to remember that just because you use free software doesn't mean
+>
+> — John Cook, [talking](http://politburo.gawker.com/introducing-the-gawker-media-securedrop-1688075673) about why Gawker set up a SecureDrop instance.
+
+Protecting the communications between journalists and sources is one of the most important press freedom issues of the 21st century. A record number of whistleblowers have recently been prosecuted in large part because the government thinks it can obtain the email and phone call records of any interaction, without ever attempting to force journalists to testify against their sources in court. A [report](https://www.aclu.org/sites/default/files/field_document/dem14-withlibertytomonitorall-07282014.pdf) from the American Civil Liberties Union, titled "With Liberty To Monitor All: How Large-Scale US Surveillance is Harming Journalism, Law And American Democracy" pointed out the numerous ways in which the US surveillance apparatus is chilling journalism. In interviews with numerous reporters, several themes emerged: 
+
+- The landscape for journalists reporting on policy failures and military decisions has been chilled and toughened.
+- Journalists believe sources are less likely to talk to them, since technology makes it much easier to follow a leak trail than before.
+- Journalists are concerned that the government now has access to "a record of everywhere I’ve walked, everywhere I’ve been." 
+
+To that end, Freedom of the Press Foundation has supported a variety of encryption tools for journalists that aim at protecting these digital communications. We hope to train journalism organizations in how to use these tools, so they can keep their sources, and themselves, safer.
+
+Once it's clear that journalists have to protect themselves from a variety of threats, not only three-letter government agencies, the next thing to consider is what steps to take to protect themselves from other threats. This line of thinking is called "operational security," or "opsec" for short.
+
+### Operational Security
+
+The NSA isn't the only adversary journalists have to be wary of. Imagine a newsroom that relies on a cloud-based storage system for documents. You and your colleagues are working on a story over the course of a few weeks. Just as you're beginning to edit the final draft, a reporter is fired. He returns his work laptop but nobody thinks to revoke his access to the cloud storage system your newsroom uses. He takes the story and the months of research to a rival newspaper and scoops you.
+
+What steps could have been taken to stop this from happening? Working through this scenario will go a long way towards threat modeling, meaning we figure out the threats facing us and decide how best to defend against them. This starts by asking questions such as:
```

Comment:
if we're going to advise using the "asset threat model" approach, you should use Jonathan Stray's guidelines, and be sure to link properly